### PR TITLE
feat(svg): add SVGs for the bricks

### PR DESCRIPTION
**Pull request by niloysikdar**
Add SVGs for the get `SVG` method for the different Brick Classes

### CODE REVIEW

File: modules/code-builder/src/brick/design0/BrickBlock.ts [hunk 1]
```diff
@@ -52,6 +52,55 @@ export default class BrickBlock extends BrickModelBlock {
...
-        return '';
+        const argsLength = Object.keys(this._args).length;
+        const repeatPath = '21 '.repeat(argsLength - 1);
+
+        return `<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+                <svg
+                width="114"
+                height="129"
+                version="1.1"
+                id="svg${this._uuid}"
```
Comment (niloysikdar) [inline modules/code-builder/src/brick/design0/BrickBlock.ts]: Should we insert ids for the different tags (i.e. svg, defs, g, g, text, etc.) like this or do you have some other thoughts @meganindya ?

File: modules/code-builder/src/brick/design0/BrickExpression.ts [hunk 2]
```diff
@@ -51,6 +51,34 @@ export default class BrickExpression extends BrickModelExpression {
     }
 
     public get SVG(): string {
-        return '';
+        return `<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (niloysikdar) [inline modules/code-builder/src/brick/design0/BrickExpression.ts:44]: Was confused about this part, can you please confirm once @meganindya ?

File: modules/code-builder/src/brick/design0/BrickStatement.ts [hunk 3]
```diff
@@ -52,6 +52,51 @@ export default class BrickStatement extends BrickModelStatement {
...
-        return '';
+        const argsLength = Object.keys(this._args).length;
+        const repeatPath = 'v 17 h -4 v -3 h -4 v 10 h 4 v -3 h 4 '.repeat(argsLength - 1);
+
+        return `<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+                <svg
+                width="85.5"
+                height="66"
+                version="1.1"
+                id="svg474"
```
Comment (niloysikdar) [inline modules/code-builder/src/brick/design0/BrickStatement.ts]: We can replace these hardcoded ids with a combination of `tag-type` and `uuid`, should I do that?

**Review by niloysikdar — COMMENTED** [review]
Looking forward to the reviews and will do the further changes

cc @meganindya

File: modules/code-builder/src/brick/design0/BrickExpression.ts (hunk 2 above)
Comment (meganindya) [reply modules/code-builder/src/brick/design0/BrickExpression.ts:44]: Don't need it I think.

File: modules/code-builder/src/brick/design0/BrickStatement.ts (hunk 3 above)
Comment (meganindya) [reply modules/code-builder/src/brick/design0/BrickStatement.ts]: ignore these for now ... design aspects are priority

File: modules/code-builder/src/brick/design0/BrickBlock.ts (hunk 1 above)
Comment (meganindya) [reply modules/code-builder/src/brick/design0/BrickBlock.ts]: ignore IDs for now

**Comment by meganindya** [comment]
Added storybook configurations ... and rebased this branch.
I think you can make the changes based off of arguments passed to the storybook component.
There's work to do in all SVGs.

**Review by meganindya — CHANGES_REQUESTED** [review]
Screenshot 2023-06-24 at 1 14 20 AM

Gotta fix the SVGs.

File: modules/code-builder/src/brick/design0/BrickBlock.ts (hunk 1 above)
Comment (niloysikdar) [reply modules/code-builder/src/brick/design0/BrickBlock.ts]: On it

File: modules/code-builder/src/brick/design0/BrickExpression.ts (hunk 2 above)
Comment (niloysikdar) [reply modules/code-builder/src/brick/design0/BrickExpression.ts:44]: I was talking about the actual SVG part; if that's the correct one!

**Pull request closed without merging**